suite-native/module-settings/src/components/CoinsSettings.tsx

```diff
@@ -45,15 +47,15 @@ const dummyCoins: { title: string; iconName: CryptoIconName; description?: strin
 
 export const CoinsSettings = () => {
     const handleCoinSelect = () => {
-        console.log('Coin select');
+        console.warn('Coin select');
```

Comment:
no warn please, it doesn't make sense in this context

Reply:
https://github.com/trezor/trezor-suite/pull/5751/commits/1a3fc9a82c5d856cae3c4f42bd5f332ab8e43328